parser-typechecker/src/Unison/Runtime/ANF.hs

```diff
@@ -169,7 +169,7 @@ expandSimple keep (v, bnd) = (v, apps' (var a v) evs)
     evs = map (var a) . Set.toList $ Set.difference fvs keep
 
 abstract :: (Var v) => Set v -> Term v a -> Term v a
-abstract keep bnd = lam' a evs bnd
```

Comment:
In the runtime we don't typically care about annotations, it's fine if we use the top-level annotation for arguments in lamdas (like we already do)